StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -387,14 +387,14 @@
       <path value="MedicationRequest.performer" />
       <type>
         <code value="Reference" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/NHSDigital-Reference" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/Reference" />
```

Comment:
Why is this profiled to point to reference as opposed to a reference to some resource?  Does this even need to be here if no profile is needed?

Reply:
DA decision was: https://fhir.nhs.uk/ResourceType/England-NAME